rally_ovs/plugins/ovs/scenarios/ovn_network.py

```diff
@@ -56,14 +56,24 @@ def create_routers_bind_ports(self, router_create_args=None,
                                   ports_per_network=None,
                                   port_bind_args=None):
 
-        # Create routers and logical networks, and connect them
```

Comment:
Instead of modifying create_routers_bind_ports scenario, it is better to keep it but modify the create_and_bind_ports scenario for this purpose, because the purpose is to create and bind ports in existing lswitches, and have nothing to do with lrouters.

The arg "use_existing_routers" can be changed to "use_existing_lswitches" as port_bind_args instead of router_create_args.

Reply:
@hzhou8 I think it's fine because are just addressing overlay use case here.  We dont want to confuse bridge workloads here

Reply:
It is not about bridge/overlay. The function create_and_bind_port is generic for both use cases, it just doesn't create lrouters. So it's better to avoid tight couple it to specific assumptions.